src/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GhDampingFunction.hpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines base-class GhDampingFunction.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+/// \ingroup GhDampingFunctionsGroup
+/// Holds classes implementing GhDampingFunction (functions \f$R^n \to R\f$).
+namespace GhDampingFunctions {
+template <size_t VolumeDim, typename Fr>
+class GaussianPlusConstant;
+}  // namespace GhDampingFunctions
+
+/*!
+ * \ingroup GhDampingFunctionsGroup
+ * Encodes a function \f$R^n \to R\f$ where n is `VolumeDim` that represents
+ * a generalized-harmonic constraint-damping parameter (i.e., Gamma0,
+ * Gamma1, or Gamma2).
+ */
+template <size_t VolumeDim, typename Fr>
+class GhDampingFunction;
```

Comment:
I'm again confused by this forward declaration. I don't think you need it

Reply:
You're right...this was left over from when we handled the 1D case separately, but when I don't have multiple specializations, this reduced to an unneeded forward declaration. Thanks for the catch!